server/odc-service/src/main/java/com/oceanbase/odc/metadb/iam/PermissionEntity.java

```diff
@@ -45,6 +48,7 @@
 @Data
 @Entity
 @EqualsAndHashCode(exclude = {"updateTime", "createTime"})
+@Where(clause = "expire_time is null or expire_time > now()")
```

Comment:
Will it affect the global settings? What should I do if I just want to query all permissions

Reply:
It will affect all query except for using customized SQL. If you want to query all permissions without any conditions, please use `findAllNoCareExpireTime()` or other customized methods.